index.js

```diff
@@ -1,19 +1,19 @@
-'use strict'
+'use strict';
 const { dialogflow, Suggestions } = require('actions-on-google');
 const express = require('express');
 const bodyParser = require('body-parser');
 const CircuitClient = require('./circuitClient');
 
 // Client ID for IMPLICIT app on Circuit. Same ID needs to be defined
 // in Account Linking of your project at console.actions.google.com
-const CLIENT_ID = 'd34edad8cda6433bb062f0671f58c232';
+const CLIENT_ID = '<your client_id>';
```

Comment:
Let's leave the client ID in here for now. Or did you change to load it dynamically from a config file?

Reply:
I thought I was supposed to hide it whenever I pushed then realized I didn't have to. Forgot to change back.